_build/server/routes/product-routes.js

```diff
@@ -23,8 +23,10 @@ var _auth2 = _interopRequireDefault(_auth);
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

Comment:
I noticed this function used in several places. Can you abstract it into an helper file and call it from other files. Ensure you use es6 arrow functions